### PR TITLE
转为动态数据

### PR DESCRIPTION
**Pull request by Jinjiang**

### CODE REVIEW

**Comment by zqzhang** [comment]
Good, @Jinjiang 

2 concerns:
- Depends on http://cdnjs.cloudflare.com/ajax/libs/vue/0.10.6/vue.min.js; can make a copy in this repo
- Dynamic data; any performance issues for multiple users access?

**Comment by Jinjiang** [comment]
@zqzhang
- No.2 is OK
- No.1: I will make a copy of code, not only this js file but also other third-party libs in the future.

**Comment by Jinjiang** [comment]
@zqzhang No. 1 is finished through `npm install` and I also changed the way to build static html file.

**Comment by zqzhang** [comment]
OK, I will check it soon.

File: README.md [hunk 1]
```diff
@@ -2,3 +2,21 @@ W3C Glossary Translation
...
 
 本專案緣起：http://lists.w3.org/Archives/Public/public-html-ig-zh/2011Mar/0160.html
+
+## 维护办法
+
+### 准备工作
+
+1. 把这个项目 `fork` 到个人的空间
+2. 把个人空间的项目克隆到本地
+3. 运行 `npm install` 安装必要的库
```
Comment (zqzhang) [inline README.md:12]: Got an error here, `Error: ENOENT, state 'C:\Users\susan\AppData\Roaming\npm'`

Fixed by making directory `C:\Users\susan\AppData\Roaming\npm`; see http://zhidao.baidu.com/link?url=ct4SoPavgKljAjHmiIGWZsv179415VrY0x8L5baTS8ryUXOKc5MwTzyIXyxAnC14s7FpE53eapyLgzeFRU1wDdgX6EFQ7TNIEjzpznkYKdm

File: README.md [hunk 2]
```diff
@@ -2,3 +2,21 @@ W3C Glossary Translation
...
+
+### 修改词汇表
+
+打开 `data` 目录下的 `glossary.json` 文件，增/删/改相应的字段。
+
+### 生成页面
+
+**提交之前千万记得**运行 `npm run build` 即可更新 `index.html`。
```
Comment (zqzhang) [inline README.md:20]: 即使没有更改 `glossary.json`，运行该命令，也会更新 `index.html`

Win7 平台

**Comment by zqzhang** [comment]
Merge as-is for further development.

File: README.md (hunk 1 above)
Comment (Jinjiang) [inline README.md:12]: 看上去这个问题所有npm使用都会重现，应该是Windows版Node.js的安装/配置问题，和我们repo的使用方式没有直接关系。
我们可以在README中加一个准备工作，就是确保node和npm都能正常运行，即可。

File: README.md (hunk 2 above)
Comment (Jinjiang) [inline README.md:20]: 不好意思，我检查了一下，`glossary.json` 写成了 `glossary.js`，粗心了
见 #5

**Pull request merged**